.github/workflows/tests.yaml

```diff
@@ -0,0 +1,48 @@
+name: Auto-tests
+
+on: [pull_request, push]
```

Comment:
Not running for some reason. Maybe rename it `.yml` instead of `.yaml`?

Probably don't need `push`. If we do however, then use
```yaml
on:
  pull_request:
  push:
    branches: [master]
```

Reply:
Thanks for casting your eye over this - You've got a good deal more experience of this than me at this point...

It won't run on master until the workflow definition is merged to master, surely?

I want to delete the `push`. It's on my to-do list at the top, and is only there for testing purposes.

Reply:
With `pull_request` only, it should run whenever you push to the PR branch. See https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request

> By default, a workflow only runs when a pull_request's activity type is `opened`, **`synchronize`**, or `reopened`

(emphasis mine)

Reply:
```suggestion
on: [pull_request]
```